docs/source/changelog.rst

```diff
@@ -1,5 +1,9 @@
 Changelog
 =========
+v1.7.4 (2020-04-02)
```

Comment:
This wouldn't be a release of 1.7.4

Reply:
There are a couple of trivial code changes from what I started with, so we can't really call it 1.7.3
(which already existed in the changelog, if not in debian packages).  I didn't see that 1.7.4 was taken on a branch somewhere.  If it works, I suppose we could change the version to 1.7.3.1.

Reply:
1.7.3.1 make sense to me.